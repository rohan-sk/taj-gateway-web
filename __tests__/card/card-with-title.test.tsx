import { render } from "@testing-library/react";
import CardWithTitle from "../../components/card/card-with-title.component";

it("shows card with title component", () => {
  const { asFragment } = render(<CardWithTitle />);
  expect(asFragment()).toMatchSnapshot();
});
