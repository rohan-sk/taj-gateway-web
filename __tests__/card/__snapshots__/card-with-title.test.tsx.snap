// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows card with title component 1`] = `<DocumentFragment />`;
